test/com/company/AcceptanceTest.java

```diff
@@ -27,8 +28,12 @@ public void helloWorldAcceptanceTest() throws IOException {
         var bytes = connection.getInputStream().readAllBytes();
         var response = new String(bytes);
 
+        var expectedRegex = Pattern.compile("Hello Bianca - the time on the server is [0-9]{1,2}:[0-9]{2}(am|pm) on [0-3]?[0-9] \\w+ [0-9]{4}");
```

Comment:
I'm glad you came up with a way to test this. This is _maybe_ ok because this is an acceptance test, but in our unit tests we should test more exactly, see my other comment

Reply:
This was my addition. like you've said for acceptance all we care is that the format is as specified by the business requirements :)